example_group.toml

```diff
@@ -0,0 +1,19 @@
+pacman = ["pacdef", { package = "pacdef", optional_deps = ["git"] }]
+yay = ["pacdef", { package = "pacdef", optional_deps = ["git"] }]
+paru = ["pacdef", { package = "pacdef", optional_deps = ["git"] }]
```

Comment:
Is this intentional? Because package duplication would be a massive problem for anyone who uses AUR helpers since they almost solely use one AUR helper with no dependence on pacman even. It'd be a major problem to see packages repeated thrice.

Reply:
It gets detected even when I have no yay binary on my system and the packages are shown as unmanaged by yay.

Reply:
Yeah this is definitely a problem, I opened #16 to discuss it.

Reply:
This is now back to how it was done originally with a config for picking which arch package manager to use for all functionality.